python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -281,9 +281,13 @@ def test_groupby_multiindex_reset_index(npartitions):
     pddf = dd.from_pandas(df.to_pandas(), npartitions=npartitions)
     gr = ddf.groupby(["a", "c"]).agg({"b": ["count"]}).reset_index()
     pr = pddf.groupby(["a", "c"]).agg({"b": ["count"]}).reset_index()
+
+    # CuDF uses "int32" for count. Pandas uses "int64"
+    gr_out = gr.compute().sort_values(by=["a", "c"]).reset_index(drop=True)
+    gr_out[("b", "count")] = gr_out[("b", "count")].astype("int64")
```

Comment:
Not sure if there is a good reason to enable (by default or optionally) the "int32" -> "int64" conversion in the `dask_cudf` aggregation itself?

Reply:
I think the reason for cudf using `int32` is that our max dataframe size is int32 rows currently. It hasn't caused issues thus far where I'd say we leave the aggregation as is.